client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -33,7 +36,9 @@ import {
 	isPristine,
 	userCanEditSettings,
 	userCanManagePayments,
-} from '../../state/label-settings/selectors';
+} from 'woocommerce/woocommerce-services/state/label-settings/selectors';
+import QueryStoredCards from 'components/data/query-stored-cards';
```

Comment:
What's the point of this if we're just going to fall back to the ones returned from the WordPress site? If we want to gradually remove custom REST endpoints and use more and more of the surrounding Calypso state, then why fall back to the WCS-returned credit cards at all?

Reply:
In wp-admin, the stored card fetch results in a 403 (but the call can be avoided by initializing `state.storedCards.isFetching` to `true`). The solution here is a short- or medium-term non-wp-admin-breaking step towards using the wpcom API-loaded state. It also has the advantage of not having to wait for it separately from the label state, though I think that's a sacrifice we've accepted in other cases. Open to other ideas for how to go about this.

Reply:
We'll have to make more decisions like this eventually. I think the way to go is to make everything work better in Calypso, and then use all kinds of tricks and shims (`calypso-stubs`, stubbing parts of the state tree, intercepting `wpcom-http` calls, etc) to make it work in WP-Admin. It's better to have just dirty tricks in one part of the equation.

I'm fine tabling this discussion for now, this is fine as it is.